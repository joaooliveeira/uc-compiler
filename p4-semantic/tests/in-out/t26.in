int main(int argc) {
  int arr = 1.0;
  return 0;
}
