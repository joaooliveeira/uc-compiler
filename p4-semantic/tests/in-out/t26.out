SemanticError: arr initialization type mismatch @ 2:7
